python_modules/dagster/dagster/_core/definitions/asset_selection.py

```diff
@@ -97,6 +97,24 @@ def keys(*asset_keys: CoercibleToAssetKey) -> "KeysAssetSelection":
         ]
         return KeysAssetSelection(*_asset_keys)
 
+    @public
+    @staticmethod
+    def key_prefixes(*key_prefixes: CoercibleToAssetKey) -> "KeyPrefixesAssetSelection":
```

Comment:
This should be CoercibleToAssetKeyPrefix, right?

Reply:
huh, didn't know that existed